config/locales/en/components/find/no_results_component.yml

```diff
@@ -0,0 +1,7 @@
+en:
+  find:
+    results:
+      no_results_component:
+        undergraduate:
+          message_html: There are not many teacher degree apprenticeship (TDA) courses on the service at the moment. You can try again soon when there may be more courses, or get in touch with us at %{contact}.
```

Comment:
It'd be good to get eyes on this from Content designers 👌

Reply:
All good and confirmed. I've asked earlier ✌️